homeassistant/components/vicare/strings.json

```diff
@@ -28,6 +28,266 @@
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
   },
+  "entity": {
+    "binary_sensor": {
+      "circulation_pump": {
+        "name": "Circulation pump"
+      },
+      "frost_protection": {
+        "name": "Frost protection"
+      },
+      "burner": {
+        "name": "Burner"
+      },
+      "compressor": {
+        "name": "Compressor"
+      },
+      "solar_pump": {
+        "name": "Solar pump"
+      },
+      "domestic_hot_water_charging": {
+        "name": "Domestic hot water charging"
+      },
+      "domestic_hot_water_circulation_pump": {
+        "name": "Domestic hot water circulation pump"
+      },
+      "domestic_hot_water_pump": {
+        "name": "Domestic hot water pump"
+      }
+    },
+    "button": {
+      "activate_onetimecharge": {
+        "name": "Activate one-time charge"
+      }
+    },
+    "climate": {
+      "heating": {
+        "name": "Heating"
+      }
+    },
+    "number": {
+      "heating_curve_shift": {
+        "name": " Heating curve shift"
+      },
+      "heating_curve_slope": {
+        "name": " Heating curve slope"
+      },
+      "normal_temperature": {
+        "name": "Normal temperature"
+      },
+      "reduced_temperature": {
+        "name": "Reduced temperature"
+      },
+      "comfort_temperature": {
+        "name": "Comfort temperature"
+      },
+      "eco_temperature": {
+        "name": "Eco temperature"
+      }
+    },
+    "sensor": {
+      "outside_temperature": {
+        "name": "Outside Temperature"
+      },
+      "return_temperature": {
+        "name": "Return Temperature"
+      },
+      "boiler_temperature": {
+        "name": "Boiler Temperature"
+      },
+      "boiler_supply_temperature": {
+        "name": "Boiler Supply Temperature"
+      },
+      "primary_circuit_supply_temperature": {
+        "name": "Primary Circuit Supply Temperature"
+      },
+      "primary_circuit_return_temperature": {
+        "name": "Primary Circuit  Return Temperature"
+      },
+      "secondary_circuit_supply_temperature": {
+        "name": "Secondary Circuit Supply Temperature"
+      },
+      "secondary_circuit_return_temperature": {
+        "name": "Secondary Circuit Return Temperature"
+      },
+      "hotwater_out_temperature": {
+        "name": "Hot Water Out Temperature"
```

Comment:
This should probably be `Domestic hot water` everywhere.

Reply:
Or just DHW?

Reply:
Is it fine to use abbreviations? For me as a non-native speaker I had to look it up, but this would be solved anyway with translations I guess.